src/components/Head.tsx

```diff
@@ -0,0 +1,15 @@
+import NextHead from 'next/head';
+import React from 'react';
+
+interface OwnProps {
+  title?: string;
+  description?: string;
+}
+
+export const Head = ({ title, description }: OwnProps) => (
+  <NextHead>
+    <title>{title || 'Power Guard Monitor'}</title>
+    <meta name="description" content={description || 'Power Guard Plant Monitoring Site'} />
+    <link rel="icon" href="/favicon.ico" />
```

Comment:
Please let me know if you want to display a different title or description.

Also, took the liberty to make a favicon with the same image as this repository. 
![image](https://user-images.githubusercontent.com/42070238/108393148-bb4be800-7256-11eb-9661-1553c34db349.png)

Reply:
Everything looks good.